content/en/continuous_integration/setup_tests/swift.md

```diff
@@ -63,7 +74,7 @@ There are two ways of installing the testing framework:
 
 ### Configuring Datadog
 
-To enable testing instrumentation, add the following environment variables to your test target. You must also select your main target in `Expand variables based on` or `Target for Variable Expansion` if using test plans:
+To enable testing instrumentation, add the following environment variables to your test target [(*)](#Alternative-Configuration). You must also select your main target in `Expand variables based on` or `Target for Variable Expansion` if using test plans:
```

Comment:
I'm not convinced about having an asterisk here. What about this? Maybe the docs team can also propose better alternatives.

```suggestion
To enable testing instrumentation, add the following environment variables to your test target (or in the `Info.plist` file as [described below](#alternative-configuration)). You must also select your main target in `Expand variables based on` or `Target for Variable Expansion` if using test plans:
```

Reply:
I have changed to your option, its more verbose but clearer